download-episodes.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python3
+import argparse
+import json
+import os
+import sys
+from argparse import RawTextHelpFormatter
+from configparser import ConfigParser
+from datetime import datetime
+from glob import glob
+from urllib import request
+
+INSTRUCTIONS = '''
+This can be used to download latest episodes from stations belonging to 
+global radio network. 
+'''
+
+DOWNLOAD_PATH = '~/Downloads'
+
+
+class Episode:
+    def __init__(self, episode):
+        self.id = episode['id']
+        self.start_date = episode['startDate']
+        self.date = datetime.strptime(episode['startDate'][0:10], '%Y-%m-%d')
+        self.stream_url = episode['streamUrl']
+        self.title = episode['title']
+        self.title_with_date = episode['titleWithDate']
+
+    def print(self):
```

Comment:
Normally you'd override `__str__` (similar to Java `toString()`) rather than adding a method to print like this.